sdk/maps/azure-maps-elevation/src/main/java/com/azure/maps/elevation/ElevationAsyncClient.java

```diff
@@ -0,0 +1,443 @@
+     * pipeline delimited string in a URL GET request. If you intend to pass more than 100 coordinates as a pipeline
+     * delimited string, use the [POST Data For
+     * Points](https://docs.microsoft.com/rest/api/maps/elevation/postdataforpoints).
+     *
+     * <p>The result will be in the same sequence of points listed in the request.
+     *
+     * @param format Desired format of the response. Only `json` format is supported.
+     * @param points The string representation of a list of points. A point is defined in lon/lat WGS84 coordinate
+     *     reference system format. If multiple points are requested, each of the points in a list should be separated
+     *     by the pipe ('|') character. The maximum number of points that can be requested in a single request is 2,000.
+     *     The resolution of the elevation data will be the highest for a single point and will decrease if multiple
+     *     points are spread further apart.
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response from a successful Get Data for Bounding Box API.
+     */
+    Mono<Response<ElevationResult>> getDataForPointsWithResponse(List<GeoPosition> points, Context context) {
+        if (points.size() < ELEVATION_DATA_SMALL_SIZE) {
+            return this.serviceClient.getDataForPointsWithResponseAsync(JsonFormat.JSON, Utility.geoPositionToString(points), context);
+        } 
+        return this.serviceClient.postDataForPointsWithResponseAsync(JsonFormat.JSON, Utility.toLatLongPairAbbreviated(points), context).onErrorMap(throwable -> {
+            if (!(throwable instanceof ErrorResponseException)) {
+                return throwable;
+            }
+            ErrorResponseException exception = (ErrorResponseException) throwable;
+            return new HttpResponseException(exception.getMessage(), exception.getResponse());
+        });
+    }
+
+    /**
+     * Get Data For Polyline
+     * <!-- src_embed com.azure.maps.elevation.async.get_data_for_polyline -->
+     * <pre>
+     * asyncClient.getDataForPolyline&#40;Arrays.asList&#40;
+     *     new GeoPosition&#40;-121.66853362143818, 46.84646479863713&#41;, 
+     *     new GeoPosition&#40;-121.65853362143818, 46.85646479863713&#41;&#41;, 5&#41;;
+     * </pre>
+     * <!-- end com.azure.maps.elevation.async.get_data_for_polyline -->
+     * 
+     * **Applies to**: S1 pricing tier.
+     *
+     * <p>The Get Data for Polyline API provides elevation data along a polyline.
+     *
+     * <p>A polyline is defined by passing in between 2 and N endpoint coordinates separated by a pipe ('|') character.
```

Comment:
> separated by a pipe ('|') character 

does this apply to Java API?

Reply:
Yes, this is how the request appears after it is processed which can be seen in postman

Reply:
Please update javadocs by GA - they should describe Java API surface and not REST APIs.